### PR TITLE
Drasi explorer reference doc update

### PR DESCRIPTION
**Pull request by NValsan**
Added description of new Debug, Attach, and Apply capabilities

### CODE REVIEW

File: docs/content/reference/developer-tools/vscode-extension/DrasiExplorerApply.png
Comment (danielgerlag) [inline docs/content/reference/developer-tools/vscode-extension/DrasiExplorerApply.png:1]: I think we should retake these screen shots with the Activity bar on the left, where most people would see it.

File: docs/content/reference/developer-tools/vscode-extension/DrasiExplorerDebug.png
Comment (danielgerlag) [inline docs/content/reference/developer-tools/vscode-extension/DrasiExplorerDebug.png:1]: We should also show these actions in the Code Lens buttons that appear above the YAML in the editor.

**Comment by agentofreality** [comment]
I dont think this is a Reference section article, I am thinking it should be in the How To section.

File: docs/content/reference/developer-tools/vscode-extension/_index.md
Comment (agentofreality) [inline docs/content/reference/developer-tools/vscode-extension/_index.md:1]: I dont think this is a Reference section article, I am thinking it should be in the How To section.

It also doesn't describe the full functionality of the extension, there are 3 sections. What is the purpose / functionality of each one and how do I use it?

**Review by agentofreality — CHANGES_REQUESTED** [review]

**Comment by NValsan** [comment]
Closing to move doc to tutorial section and make requested changes

**Pull request closed without merging**